src/Data/Food/Builder/Recipe.elm

```diff
@@ -360,6 +414,21 @@ computeIngredientImpacts ({ mass } as recipeIngredient) =
         |> Impact.mapImpacts (computeImpact mass)
 
 
+computeIngredientsTotalBonus : List Impact.Definition -> List RecipeIngredient -> Unit.Impact
+computeIngredientsTotalBonus defs =
+    List.foldl
+        (\({ bonuses } as recipeIngredient) acc ->
+            recipeIngredient
+                |> computeIngredientImpacts
+                |> computeIngredientBonusesImpacts defs bonuses
+                |> .total
+                |> Unit.impactToFloat
```

Comment:
Nit: Having a `Unit.add` helper would also avoid the unwrapping/rewrapping here

Reply:
I went with `Quantity.plus`